test/version_commands.bats

```diff
@@ -341,9 +341,9 @@ teardown() {
 }
 
 @test "global should preserve symlinks when setting versions" {
-  mkdir $HOME/other-dir
-  touch $HOME/other-dir/.tool-versions
```

Comment:
Is this not required for the test?

Reply:
Oops I'm not sure how this snuck in - fixed